README.md

```diff
@@ -1,99 +1,66 @@
-|--------------------------------------------------------------------|:---------------------:|-------|
-| :white_check_mark: `get_block_number_by_timestamp`                 |  :white_check_mark:   |       |
-| :white_check_mark: `get_current_value`                             |  :white_check_mark:   |       |
-| :white_check_mark: `get_data_before`                               |  :white_check_mark:   |       |
-| :white_check_mark: `get_new_value_count_by_query_id`               |  :white_check_mark:   |       |
-| ~~`get_pending_reward_by_staker`~~                                 |                       |       |
-| ~~`get_real_staking_rewards_balance`~~                             |                       |       |
-| :white_check_mark: `get_report_details`                            |  :white_check_mark:   |       |
-| :white_check_mark: `get_reporter_by_timestamp`                     |  :white_check_mark:   |       |
-| :white_check_mark: `get_reporter_last_timestamp`                   |  :white_check_mark:   |       |
-| :white_check_mark: `get_reporting_lock`                            |  :white_check_mark:   |       |
-| :white_check_mark: `get_reports_submitted_by_address`              |  :white_check_mark:   |       |
-| :white_check_mark: `get_reports_submitted_by_address_and_query_id` |  :white_check_mark:   |       |
-| :white_check_mark: `get_stake_amount`                              |  :white_check_mark:   |       |
-| :white_check_mark: `get_staker_info`                               |  :white_check_mark:   |       |
-| :white_check_mark: `get_time_of_last_new_value`                    |  :white_check_mark:   |       |
-| :white_check_mark: `get_timestamp_by_query_id_and_index`           |  :white_check_mark:   |       |
-| :white_check_mark: `get_index_for_data_before`                     |  :white_check_mark:   |       |
-| :white_check_mark: `get_timestamp_index_by_timestamp`              |  :white_check_mark:   |       |
-| :white_check_mark: `get_total_stake_amount`                        |  :white_check_mark:   |       |
-| :white_check_mark: `get_total_stakers`                             |  :white_check_mark:   |       |
-| :white_square_button: `get_total_time_based_rewards_balance`?      | :white_square_button: |       |
-| :white_check_mark: `is_in_dispute`                                 |  :white_check_mark:   |       |
-| :white_check_mark: `retrieve_data`                                 |  :white_check_mark:   |       |
+### Dispatchable Functions
 
-### Governance
+#### For Users
+- `add_staking_rewards` - Funds the pallet with staking rewards, which can be used to incentivize oracle usage.
+- `begin_dispute` - Initialises a dispute/vote in the system. Requires a dispute fee to be paid.
+- `fund_feed` - Allows a data feed to be funded with tokens.
+- `send_votes` - Sends any dispute votes due to be sent to the governance controller contract for tallying.
```

Comment:
Might want to clarify that it's able to send votes while the voting period hasn't elapsed, and that it doesn't tally the votes.

Reply:
Updated send_votes to the following:

"Sends any dispute votes to the governance controller contract for tallying, provided the voting period hasn't elapsed."